apps/omg/lib/omg/state/core.ex

```diff
@@ -41,7 +70,10 @@ defmodule OMG.State.Core do
           tx_index: non_neg_integer(),
           # NOTE: that this list is being build reverse, in some cases it may matter. It is reversed just before
           #       it leaves this module in `form_block/3`
-          utxo_db_updates: list(db_update())
+          utxo_db_updates: list(db_update()),
+          # NOTE: because UTXO set is not loaded from DB entirely, we need to remember the UTXOs spent in already
+          # processed transaction before they get removed from DB on form_block.
+          recently_spent: MapSet.t(Utxo.Position.t())
```

Comment:
shouldn't be `(InputPointer.Protocol.t)` here?

Reply:
:+1: